src/main/java/uk/gov/hmcts/ccd/cache/CachingConfiguration.java

```diff
@@ -35,13 +46,14 @@ private void configCaches(int definitionCacheTTL, Config config) {
         config.addMapConfig(newMapConfig("caseTabCollectionCache", definitionCacheTTL));
         config.addMapConfig(newMapConfig("wizardPageCollectionCache", definitionCacheTTL));
         config.addMapConfig(newMapConfig("userRolesCache", definitionCacheTTL));
+        cacheWarmUpService.warmUp();
```

Comment:
the ticket also mentions to add the ability to enable/disable the pre warm up per environment. Can we add that please? 
I might be wrong but I don't see it. 
For local development better to have it disabled. Initially we will enable this on Demo only. If all looks good we'll then enable on AAT and Prod

Reply:
Is this because you do not want to hit def store db in some envs, correct? I would imagine for local dev it does not really matter. Remember it is fail safe so it won't stop the app starting up. But yes if that is comprehended and still the performance is a concern then I can add it per env.

Reply:
I have added the flag anyway. Better to have more control as you say 👍